### PR TITLE
Package properties checks

### PR DESCRIPTION
**Pull request by tisnik**

### CODE REVIEW

File: integration-tests/features/src/attribute_checks.py [hunk 1]
```diff
@@ -159,3 +159,37 @@ def get_details_node(context):
...
+
+    year = int(re.fullmatch(pattern, cve).group(1))
+    current_year = datetime.datetime.now().year
+
+    # well the lower limit is a bit arbitrary
+    # (according to SRT guys it should be 1999)
+    assert year >= 1999 and year <= current_year
+
+
+def check_cve_value_with_score(cve):
```
Comment (jpopelka) [inline integration-tests/features/src/attribute_checks.py]: The function is almost the same as `check_cve_value()`.
The only difference is AFAICT the `pattern`, so maybe you could just add a boolean `has_score` argument to the `check_cve_value()` and choose `pattern` based on the `has_score` ?

Comment (tisnik) [reply integration-tests/features/src/attribute_checks.py]: @jpopelka very good idea, I'll update the code. Thanks!

**Comment by jpopelka** [comment]
👍 feel free to merge

**Comment by tisnik** [comment]
Thanks @jpopelka

**Pull request merged**